helm/hpcc/templates/network.yaml

```diff
@@ -157,6 +157,25 @@ spec:
         matchLabels:
           app: eclservices
 ---
+{{ range .Values.esp -}}
```

Comment:
Should this walk all components that have a service and generate a NetworkPolicy if they have ingress?
So all services are covered in same way.

Apart from esp and roxie services, there's also potentially dafilesrv and sasha services we might want ingress on too.

Reply:
Agree with Jake, I don't know why this is just for esp rather than for anything defining a service

Reply:
Each component seems define service differently. Could we defer this for later improvement? I currently only implement the services through "External IP"